tests/system/test_error_details.py

```diff
@@ -47,14 +45,19 @@ def create_bad_request_details():
         field_violation.field = "test field"
         field_violation.description = "test description"
         return bad_request_details
+
     bad_request_details = create_bad_request_details()
     status = create_status(bad_request_details)
 
     with pytest.raises(exceptions.GoogleAPICallError) as e:
-        _ = echo.echo(showcase.EchoRequest(
-            error=status,
-        ))
-        assert e.details == [bad_request_details]
+        _ = echo.echo(
+            showcase.EchoRequest(
+                error=status,
+            )
+        )
+
+    # Note: error details are exposes e.value.details.
```

Comment:
```suggestion
    # Note: error details are exposed as e.value.details.
```

Also in multiple places in this file